woofbot/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_WOOFBOT_WEB_IP
+      APP_PORT: $APP_WOOFBOT_WEB_PORT
+  web:
+    image: woofbot/woofbot:v0.1.0@sha256:0a9bdae77f44ed9ce875d50fdbd94e4909d88ac77592fe04661e7d3171911ed9
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "${APP_WOOFBOT_WEB_PORT}:8080"
```

Comment:
All traffic to your app will go through the App Proxy (which is a new component in Umbrel v0.5 that automatically ensures Woofbot is password protected).

Therefore you should drop this `ports` block as traffic will go through the App Proxy and directly to the `web` container via the internal network.

Reply:
Fixed - also uploaded a new container image that will use APP_PORT (8080) as the default port.